sound/soc/intel/boards/sof_ssp_amp.c

```diff
@@ -98,7 +98,7 @@ static struct snd_soc_dai_link_component platform_component[] = {
 
 static struct snd_soc_dai_link *
 sof_card_dai_links_create(struct device *dev, enum sof_ssp_codec amp_type,
-			  int ssp_codec, int dmic_be_num, int hdmi_num,
+			  int ssp_amp, int dmic_be_num, int hdmi_num,
```

Comment:
I would push the s/codec/amp to a different patch, the renaming makes the patch a bit hard to review

Reply:
done